test/e2e/metrics_e2e_test.go

```diff
@@ -567,6 +571,7 @@ func getMetricsFromPod(client operatorclient.ClientInterface, pod *corev1.Pod) [
 				m.Value = c.GetValue()
 			}
 			metrics = append(metrics, m)
+			fmt.Printf("metrics %+v\n", m)
```

Comment:
It's unclear to me how much value we get logging the metrics. Did this help debug this test flake, or did you think it was improvement in e2e visibility? If so, can we use the `ctx.Ctx().Logf(...)` logger instead of fmt.Printf here, else we can just drop this log call entirely.

Reply:
Echoing this sentiment. I wouldn't mind if these metrics were printed on failure but otherwise these logs seem to offer little value add.

Reply:
Sorry, it was for my own debug.  I'll take it out.  Thanks!